dev.budd.Switcheroo.yml

```diff
@@ -0,0 +1,45 @@
+app-id: dev.budd.Switcheroo
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: run.sh
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.rust-stable"
+  - "org.freedesktop.Sdk.Extension.llvm16"
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--device=all"
+  - "--filesystem=host-etc"
```

Comment:
Why is `host-etc` needed?

Reply:
The program needs udev rules installed outside the sandbox so I wanted to add a way to detect if they were there (by checking /etc/) and provide instructions on how to install them if not.

this could also reasonably be done with a first time run script that they have to read since its likely no user would have the rules installed on first run.

Reply:
> The program needs udev rules installed outside the sandbox so I wanted to add a way to detect if they were there (by checking /etc/udev/) and provide instructions on how to install them if not.

You could just check the permissions instead of punching a hole in the sandbox.